src/rustllvm/PassWrapper.cpp

```diff
@@ -359,7 +359,17 @@ LLVMRustAddAlwaysInlinePass(LLVMPassManagerBuilderRef PMB, bool AddLifetimes) {
 extern "C" void
 LLVMRustRunRestrictionPass(LLVMModuleRef M, char **symbols, size_t len) {
     llvm::legacy::PassManager passes;
-    passes.add(llvm::createInternalizePass());
+
+    auto PreserveFunctions = [=](const GlobalValue &GV) {
```

Comment:
It was used in the old version, see https://github.com/rust-lang/rust/pull/34743/commits/0763041c3589c042ed6c57ad49951eb7d584f675#diff-2030f049f8df2454e3720ea9403bc14aL363.

I should have squashed that.